src/libs/dutil/thmutil.cpp

```diff
@@ -2331,6 +2336,11 @@ static HRESULT ParseControls(
         {
             type = THEME_CONTROL_TYPE_TAB;
         }
+        else if (CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"Combobox", -1) ||
+                 CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"cm", 2))
```

Comment:
Any reason to introduce a new old-school name? They're all obsolete in WiX v4.0.